test/statsCases/named-chunk-groups/webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const stats = {
+	hash: false,
+	timings: false,
+	builtAt: false,
+	assets: false,
+	chunks: true,
+	chunkOrigins: true,
+	entrypoints: false,
```

Comment:
Could you also test the combination of entrypoints and chunkGroups?

Reply:
Maybe just by exporting multiple configurations in this file

Reply:
The combination of both of them is checked through the `detailed` and `verbose` - Chunk Group is not added to the output since there is only 1 chunk group which is the entry point.
But i can add a more explicit test here